pkgs/misc/themes/elementary/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "elementary-gtk-theme-${version}";
+  version = "5.1.1";
+
+  src = fetchFromGitHub {
+    owner = "elementary";
+    repo = "stylesheet";
+    rev = "${version}";
```

Comment:
Nitpick: could also be just `rev = version;`

Reply:
changed. thanks